src/assets/json/psp.json

```diff
@@ -1,30 +1,114 @@
 [
-    {
```

Comment:
@vito80ba In my opinion, this file shouldn't be stored in assets folder. If it's just a mock we should move it in a dedicated folder